### PR TITLE
Drop ProposalObject and use Proposal model instead

### PR DESCRIPTION
**Pull request by bkutil**
Refs https://github.com/crowbar/barclamp-crowbar/pull/1269

### CODE REVIEW

**Comment by jsuchome** [comment]
+1

File: crowbar_framework/app/models/ceph_service.rb [hunk 1]
```diff
@@ -252,7 +252,7 @@ def validate_proposal_after_save proposal
     end
 
     unless radosgw_nodes.empty?
-      ProposalObject.find_proposals("swift").each {|p|
+      Proposal.where(barclamp: "swift").each {|p|
```
Comment (vuntz) [inline crowbar_framework/app/models/ceph_service.rb]: Isn't this missing a `.all`?

Comment (bkutil) [inline crowbar_framework/app/models/ceph_service.rb]: Good question. I just tested it on a REPL and it should work fine...

Comment (vuntz) [inline crowbar_framework/app/models/ceph_service.rb]: The part that confuses me is that you used `.all` in https://github.com/crowbar/barclamp-swift/pull/286/files#diff-01dcdaa5e35c14cdff38ef4940cdb8fdR308

So trying to understand which way is better :-)

Comment (bkutil) [inline crowbar_framework/app/models/ceph_service.rb]: > The part that confuses me is that you used `.all` in https://github.com/crowbar/barclamp-swift/pull/286/files#diff-01dcdaa5e35c14cdff38ef4940cdb8fdR308

Right. If I tested that before using `.all`, I would've found out select
without `.all` is also OK :)

Comment (tboerger) [inline crowbar_framework/app/models/ceph_service.rb]: You don't need all. Without all it's lazy loaded, but it doesn't matter anyway as the records are getting loaded when they are accessed like here with ```each```.

**Comment by tboerger** [comment]
:+1:

**Pull request merged**